### PR TITLE
Insert and Update APIs with their tests

### PR DESCRIPTION
**Pull request by abhijith0505**
- added /insert and /update
- added tests for /insert and /update
- added doc strings to methods functions
- tests now create a new database before execution and clears the test database after every test

### CODE REVIEW

File: MOCC/src/configurations/Comm-Layer/src/tests/test_config_manager.py [hunk 1]
```diff
@@ -1,22 +1,55 @@
...
+		print("Testing /insert")
+
+		response = self.app.get('/insert/test_config_manager_insert', follow_redirects=True)
+		assert b'Device address exist' in response.data
+
+	def test_update_path(self):
+		"""Calls the /update API to test updating of device address."""
+		print("Testing /update")
+
+		response = self.app.get('/update/test_config_manager', follow_redirects=True)
```
Comment (mtambos) [inline MOCC/src/configurations/Comm-Layer/src/tests/test_config_manager.py]: I one were to call this test alone, without running test_insert_path or test_save_path first, would it run successfully?

File: MOCC/src/configurations/Comm-Layer/src/tests/test_config_manager.py [hunk 2]
```diff
@@ -1,22 +1,55 @@
...
 import json
-import tempfile
 import sys
 sys.path.insert(0, '..')
 import config_manager
+import tango_db_helper as DBHelper
 
 class ConfigManagerTest(unittest.TestCase):
+
 	def setUp(self):
```
Comment (bohrium272) [inline MOCC/src/configurations/Comm-Layer/src/tests/test_config_manager.py:11]: `set_up`? Although I don't think this was part of your commit...

**Review by bohrium272 — CHANGES_REQUESTED** [review]

File: MOCC/src/configurations/Comm-Layer/src/tests/test_config_manager.py (hunk 2 above)
Comment (abhijith0505) [reply MOCC/src/configurations/Comm-Layer/src/tests/test_config_manager.py:11]: it is part of the commit

File: MOCC/src/configurations/Comm-Layer/src/tests/test_config_manager.py (hunk 1 above)
Comment (abhijith0505) [reply MOCC/src/configurations/Comm-Layer/src/tests/test_config_manager.py]: Yes, all tests work successfully individually now

File: MOCC/src/configurations/Comm-Layer/src/tests/test_config_manager.py (hunk 2 above)
Comment (abhijith0505) [reply MOCC/src/configurations/Comm-Layer/src/tests/test_config_manager.py:11]: setUp runs before every test to initialise the test instance of the flask app

Comment (bohrium272) [reply MOCC/src/configurations/Comm-Layer/src/tests/test_config_manager.py:11]: OK, our coding convention means it should be called `set_up`, right?

Comment (abhijith0505) [reply MOCC/src/configurations/Comm-Layer/src/tests/test_config_manager.py:11]: setUp is defined by flask directly. It's not a user defined function

**Pull request merged**